docs/source/manual/testing.rst

```diff
@@ -344,10 +332,44 @@ The ``dropwizard-testing`` module offers helper classes for your easily doing so
 The optional ``dropwizard-client`` module offers more helpers, e.g. a custom JerseyClientBuilder,
 which is aware of your application's environment.
 
-JUnit
+JUnit 5
 -----
-Adding ``DropwizardAppRule`` to your JUnit test class will start the app prior to any tests
+Adding ``DropwizardExtensionsSupport`` annotation and ``DropwizardAppExtension`` extension to your JUnit5 test class will start the app prior to any tests
 running and stop it again when they've completed (roughly equivalent to having used ``@BeforeAll`` and ``@AfterAll``).
+``DropwizardAppExtension`` also exposes the app's ``Configuration``,
+``Environment`` and the app object itself so that these can be queried by the tests.
+
+If you don't want to use the ``dropwizard-client`` module or find it excessive for testing, you can get access to
+a Jersey HTTP client by calling the `client` method on the extension. The returned client is managed by the extension
+and can be reused across tests.
+
+.. code-block:: java
+
+    @ExtendWith(DropwizardExtensionsSupport.class)
+    class LoginAcceptanceTest {
+
+        private static DropwizardAppExtension<TestConfiguration> dropwizard = new DropwizardAppExtension<>(
```

Comment:
I think the should the `dropwizard` variable here be `RULE`

Reply:
What about `EXTENSION` as in [`dropwizard-testing` module test classes](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-testing/src/test/java/io/dropwizard/testing/junit5/DropwizardAppExtensionTest.java) ?

I find `RULE` quite confusing given JUnit5 changed basically changed rule framework to extension framework. Same thing applies for tests that still uses `RULE` IMO.

By the way, I think I copied the `LoginAcceptanceTest` (and the `dropwizard` variable naming) from an existing test so we might want to modify this test as well.

Reply:
I'm fine with EXTENSION, I just wanted to ensure the code was correct